src/hypertrace/agent/__init__.py

```diff
@@ -70,6 +30,7 @@ def __new__(cls):
         if cls._instance is None:
             with cls._singleton_lock:
                 logger.debug('Creating Agent')
+                logger.debug('Python version: %s', sys.version)
```

Comment:
this is just to help when trying to debug issues(sometimes the stack traces only include the `major.minor` version.